code/datums/ert/outsider.dm

```diff
@@ -5,7 +5,7 @@
 
 /datum/responseteam/kataphracts
 	name = "Kataphracts"
-	chance = 15
+	chance = 10
```

Comment:
no bias here, but i don't think this is needed, i can review the rest of the PR if this is reverted